emr-etl-runner/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM jruby:9.1
+MAINTAINER Josh Cox <josh 'at' webhosting.coop>
+
+ENV PLAIN_OF_JARS=20170705 \
+LANG=en_US.UTF-8
+
+RUN apt-get update \
+&& apt-get install -yqq gettext-base mlocate \
```

Comment:
which I guess could be separated into another container, but I'm running them both in this container at the moment, which may not have been clear